src/components/SetFavoriteRegion.js

```diff
@@ -25,6 +26,9 @@ export default function SetFavoriteRegion() {
     } else if (user.favoriteRegions.includes(enteredRegion)) {
       setMessage('이미 선택된 관심지역입니다.');
       return;
+    } else if (user.favoriteRegions.length === 3) {
+      setMessage('관심지역의 갯수를 초과하였습니다.');
+      return;
```

Comment:
```suggestion
     if (user.favoriteRegions.includes(enteredRegion)) {
      setMessage('이미 선택된 관심지역입니다.');
      return;
     }
     
    if (user.favoriteRegions.length === 3) {
      setMessage('관심지역의 갯수를 초과하였습니다.');
      return;
    }
    
```

return 시켜버리기 때문에 else if가 꼭 있을 필요가 없습니다. else if는 꼭 써야하는 경우가 아니라면 빼주세요!